tests/integration/models/billing.spec.coffee

```diff
@@ -172,8 +181,6 @@ describe 'Billing Model', ->
 							name: 'Standard support'
 
 					m.chai.expect(plan).to.have.property('currentPeriodEndDate').that.is.a('string')
-					m.chai.expect(plan).to.have.property('id').that.is.a('string')
```

Comment:
This one is no longer present, but since we don't have it in the typings, I guess we never used it on the Dashboard